### PR TITLE
Not sure if this works

### PR DESCRIPTION
**Pull request by emordonez**
Tried to create a way for the LOOK command to check for a Room or Item argument after it. Changed the Room's items data structure to a dict for this to work. Also added unique ID's for Items, haven't implemented it into anything yet.

### CODE REVIEW

File: assets/room.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Room:
 	def __init__(self, x = 0, y = 0, name = "", items = None, description = ""):
 		if items is None:
-			items = []
+			items = {}
```
Comment (Lathie) [inline assets/room.py]: hashmap use is actually non-ideal since hashmaps technically aren't iterable. I vote we keep the list

**Comment by emordonez** [comment]
Changed the items back to a list, and output now prints the room name/description or the description of each item in the room

**Comment by nirm123** [comment]
Would it be simpler to make look room and look item separate (as in separate message codes/functions in update/output)? That way you don't need to carry around extra arguments to differentiate?

**Comment by nirm123** [comment]
Also, I'm not sure if I've misunderstood something, but I thought that look "item" was meant for getting a description for a specific item. So look "book" would give you a description of the book.

**Comment by emordonez** [comment]
I'm going to try and implement that after this gets resolved. I think if we try it the way you suggested, by separating the parsing for rooms and items, then the parser could handle item ID lookups and you can look at individual items.

**Pull request merged**